rest/available-phone-numbers/local-basic-example-8/local-get-basic-example-8.5.x.cs

```diff
@@ -0,0 +1,28 @@
+// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Rest.Api.V2010.Account.AvailablePhoneNumberCountry;
+using System.Linq;
+
+class Example
+{
+    static void Main(string[] args)
+    {
+        // Find your Account Sid and Auth Token at twilio.com/console
+        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+        const string authToken = "your_auth_token";
+        TwilioClient.Init(accountSid, authToken);
+
+        var localAvailableNumber = LocalResource.Read("US",
+                                                      excludeLocalAddressRequired: true);
+
+        // Purchase the first number on the list
+        var firstNumber = localAvailableNumber.FirstOrDefault();
```

Comment:
Coming from Java that seems a bit like using exceptions for flow control, but I think it would make the example simpler. I'll also remove the null checking